main.py

```diff
@@ -670,22 +590,62 @@ def sizechanged(e):
     manager_button.configure(width=int(app.winfo_width() / 12))
 
 
-# THEMED IMAGES
+def make_windows_blur(window):
+    from ctypes import windll
+
+    from BlurWindow.blurWindow import GlobalBlur
+
+    GlobalBlur(
+        windll.user32.GetParent(window.winfo_id()),
+        Acrylic=True,
+        Dark=(theme == "Dark"),
+        hexColor=bg_color,
+    )
+
+
+# LOAD IMAGES
+theme_dark = PhotoImage(file="./assets/images/dark/dark_theme.png")
+theme_light = PhotoImage(file="./assets/images/light/dark_theme.png")
+
+transparency_dark = PhotoImage(file="./assets/images/dark/transparency.png")
+transparency_light = PhotoImage(file="./assets/images/light/transparency.png")
+
+speaker_dark = PhotoImage(file="./assets/images/dark/speaker.png")
+speaker_light = PhotoImage(file="./assets/images/light/speaker.png")
+
+bell_dark = PhotoImage(file="./assets/images/dark/bell.png")
+bell_light = PhotoImage(file="./assets/images/light/bell.png")
+
+pin_dark = PhotoImage(file="./assets/images/dark/pin.png")
+pin_light = PhotoImage(file="./assets/images/light/pin.png")
+
+github_logo_dark = PhotoImage(file="./assets/images/dark/github.png")
+github_logo_light = PhotoImage(file="./assets/images/light/github.png")
+
+globe_dark = PhotoImage(file="./assets/images/dark/globe.png")
+globe_light = PhotoImage(file="./assets/images/light/globe.png")
+
+logo = PhotoImage(file="./assets/logo_new_150x150.png")
+
 
 if theme == "Dark":
     settings_btn.configure(image=settings_image_dark)
-    GlobalBlur(HWND, Acrylic=True, Dark=True)
 elif theme == "Light":
     settings_btn.configure(image=settings_image_light)
-    GlobalBlur(HWND, Acrylic=True, hexColor=f"{bg_color}")
     time_display.configure(fg="black")
     time_selected_display.configure(fg="black")
 
+if system() == "Windows":
+    make_windows_blur(app)

```

Comment:
Looking at the BlurWindow source-code, blur doesn't work on macOS either.

Reply:
I actually objected to this and even commented the code out but @not-nef did a nicer implementation. He said he'll add a toggle, which is what I need since my PC doesn't need blur, transparency is enough.